docs/sql-guide/statements/statement-insert-bulk-csv-example.md

```diff
@@ -73,5 +74,4 @@ SELECT TOP(10) * FROM age;
 ## Further information
 
 * [SELECT statement](/docs/sql-guide/statements/statement-select)
-* [BULK INSERT using NDJSON data source](/docs/sql-guide/statements/statement-insert-bulk-ndjson-example)
-* [BULK INSERT using PARQUET data source](/docs/sql-guide/statements/statement-insert-bulk-parquet-example)
+* [BULK INSERT statement](/docs/sql-guide/statements/statement-insert-bulk)
```

Comment:
FYI I pulled links to the other examples because it's too complicated to maintain a list of alternate pages like it had. Plus the BULK INSERT statement has everything user need, including links to the examples